docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+Within \texttt{pyuvdata}, antenna positions are treated as static, relative to some fixed point on the surface of the Earth/Moon rotating about some axis\footnote{This can drift with time due to ``polar wobble'', among other effects, which is accounted for in astrometric calculations.}. This is obviously unsuitable for antennas which are not affixed to the ground (looking at you, space VLBI aficionados), but this also turns out to be untrue even for ground base instruments due to tidal effects, which on Earth can cause displacements of up to 55 cm. For most arrays, these effects are ``common mode'' -- they affect all antennas in the same way -- but it can produce milliarcsecond-level effects which are not presently compensated for. As such, the \texttt{pyuvdata} phasing framework is likely not be suitable for VLBI applications.
 
-\begin{thebibliography}{9}
+\subsection{Relativistic $uv$-stretching}
+To first order, stellar aberration is accounted for as part of basic astrometry calculations, wherein center phasing position is calculated to very high accuracy. But there is a second-order effect, where the image plane is relativistically ``stretched'' ever so slightly, which translate in the $u$ and $v$ coordinates being marginally misaligned to the reference coordinate system (typically ICRS). The impact of annual aberration generally does not exceed one part in $10^{4}$, with maximum impact seen for fields along the ecliptic. Diurnal aberration is generally limited to one part in $10^{6}$, with maximum impact seen for equatorial targets. In both cases, the impact is only significant when dealing with an interferometer with long baselines relative to its field of view, i.e. where $|\mathbf{B}_{\rm max}| \sim \xi D_{\rm ant}$, where $\mathbf{B}_{\rm max}$ is the maximum-length baseline vector, $D_{\rm ant}$ is the antenna diameter, and $\xi$ is the impact factor of the aberration (where $\xi=10^{4}$ for annual aberration, and $\xi=10^{6}$ for diurnal aberration.
 
-\bibitem{USNO}
-  George Kaplan,
-  \textit{USNO Circular 179},
-  \href{http://aa.usno.navy.mil/publications/docs/Circular_179.php}{\url{http://aa.usno.navy.mil/publications/docs/Circular_179.php}},
-  2005
- 
- \bibitem{kaplan1998}
-  George H. Kaplan, High-Precision Algorithms for Astrometry: A Comparison of Two Approaches,
-  \textsl{The Astronomical Journal}, 115, 1, 1998
+\subsection{$w$-projection effects}
+Finally, we note $uvw$-coordinates are only calculated for a central phase center position. Wide-field interferometers -- where the spherical curvature of the sky means that patches of sky \emph{away} from the phase center will be warped by non-coplanar effects -- require additional care in handling. While pyuvdata tools provide no explicit support for handling such effects, several analysis packages (e.g., WSClean; \citealt{WSClean}) that interface with \texttt{pyuvdata}-supported formats are capable of doing so. Additionally, the orientation of the $uvw$-coordinates 
```

Comment:
Ok, if you're going to cite Offringa you need to also list FHD, citation is:

```
@ARTICLE{2012ApJ...759...17S,
       author = {{Sullivan}, I.~S. and {Morales}, M.~F. and {Hazelton}, B.~J. and {Arcus}, W. and {Barnes}, D. and {Bernardi}, G. and {Briggs}, F.~H. and {Bowman}, J.~D. and {Bunton}, J.~D. and {Cappallo}, R.~J. and {Corey}, B.~E. and {Deshpande}, A. and {deSouza}, L. and {Emrich}, D. and {Gaensler}, B.~M. and {Goeke}, R. and {Greenhill}, L.~J. and {Herne}, D. and {Hewitt}, J.~N. and {Johnston-Hollitt}, M. and {Kaplan}, D.~L. and {Kasper}, J.~C. and {Kincaid}, B.~B. and {Koenig}, R. and {Kratzenberg}, E. and {Lonsdale}, C.~J. and {Lynch}, M.~J. and {McWhirter}, S.~R. and {Mitchell}, D.~A. and {Morgan}, E. and {Oberoi}, D. and {Ord}, S.~M. and {Pathikulangara}, J. and {Prabu}, T. and {Remillard}, R.~A. and {Rogers}, A.~E.~E. and {Roshi}, A. and {Salah}, J.~E. and {Sault}, R.~J. and {Udaya Shankar}, N. and {Srivani}, K.~S. and {Stevens}, J. and {Subrahmanyan}, R. and {Tingay}, S.~J. and {Wayth}, R.~B. and {Waterson}, M. and {Webster}, R.~L. and {Whitney}, A.~R. and {Williams}, A. and {Williams}, C.~L. and {Wyithe}, J.~S.~B.},
        title = "{Fast Holographic Deconvolution: A New Technique for Precision Radio Interferometry}",
      journal = {\apj},
     keywords = {dark ages, reionization, first stars, techniques: image processing, techniques: interferometric, Astrophysics - Instrumentation and Methods for Astrophysics, Astrophysics - Cosmology and Extragalactic Astrophysics},
         year = 2012,
        month = nov,
       volume = {759},
       number = {1},
          eid = {17},
        pages = {17},
          doi = {10.1088/0004-637X/759/1/17},
archivePrefix = {arXiv},
       eprint = {1209.1653},
 primaryClass = {astro-ph.IM},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2012ApJ...759...17S},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

Reply:
Also, the last sentence in this paragraph is a fragment, not sure where it's going.

Reply:
Citation added, and whoops -- that was a comment about the fact that the orientation is only correctly set for the phase center, and may not be correct for particularly wide fields of view.